mlflow/deployments/databricks/__init__.py

```diff
@@ -141,6 +141,7 @@ def _call_endpoint(
             timeout=MLFLOW_HTTP_REQUEST_TIMEOUT.get() if timeout is None else timeout,
             raise_on_status=False,
             retry_codes=MLFLOW_DEPLOYMENT_CLIENT_REQUEST_RETRY_CODES,
+            extra_headers={"X-Databricks-Endpoints-API-Client": "EndpointsDeploymentClient"},
```

Comment:
```suggestion
            extra_headers={"X-Databricks-Endpoints-API-Client": "Endpoints Deployment Client"},
```
Actually let's add spaces